tools/android/packaging/xbmc/AndroidManifest.xml.in

```diff
@@ -9,6 +9,7 @@
         android:minSdkVersion="21"
         android:targetSdkVersion="26" />
 
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
```

Comment:
This won't fix anything.

> Any app that declares the WRITE_EXTERNAL_STORAGE permission is implicitly granted this permission.

https://developer.android.com/reference/android/Manifest.permission.html#READ_EXTERNAL_STORAGE